### ISSUE TITLE
[Bug] Invalid Maven

### ISSUE THREAD

**Issue by PikaMug** [NicoNekoDev/AstralBooks#16]
### Information
Cannot compile this repository with provided instructions using Eclipse Maven.

### Details
Following the README instructions results in the following error:

`[ERROR] 'dependencies.dependency.version' for ro.nicuch:CitizensBooks:jar must be a valid version but is '${latest.version}'.`

Replacing the version with `2.4.7` like so...

```
        <repository>
            <id>citizensbooks_repo</id>
            <url>https://raw.github.com/nicuch/maven_repo/</url>
        </repository>

        <dependency>
            <groupId>ro.nicuch</groupId>
            <artifactId>CitizensBooks</artifactId>
            <version>2.4.7</version>
            <scope>provided</scope>
        </dependency>
```

...results in the following:

`[WARNING] The POM for ro.nicuch:CitizensBooks:jar:2.4.7 is missing, no dependency information available`

My first thought is that the repository could be offline? I'm unable to compile with JitPack 
 ( https://jitpack.io/#nicuch/CitizensBooks ) but that's for a completely different reason, which is #17

**Comment by NicoNekoDev** [comment]
The repo was hosted directly on github using raw data, but github removed that feature. Sadly, unless I get an actual repo, I can't do anything.

**Comment by PikaMug** [comment]
No problem! Once the project is able to be compiled, you can just tell people to use Jitpack.

**Comment by PikaMug** [comment]
Resolved with #19
